### PR TITLE
[BACKPORT][v1.2.x] Failed backups cleanup: update deploy YAML

### PR DESCRIPTION
**Pull request by mantissahz**
Add a new option failed-backup-ttl and update the LEP for failed backup cleanup.
and modify Failed Backup cleanup LEP

https://github.com/longhorn/longhorn/issues/4468, https://github.com/longhorn/longhorn/issues/3898

Signed-off-by: James Lu [james.lu@suse.com](mailto:james.lu@suse.com)

### CODE REVIEW

File: chart/questions.yml [hunk 1]
```diff
@@ -324,6 +324,16 @@ The available volume spec options are:
     type: int
     min: 0
     default: 300
+  - variable: defaultSettings.failedBackupTTL
+    label: Failed Backup Time to Live
+    description: "In minutes. This setting determines how long Longhorn will keep the backup resource that was failed. Set to 0 to disable the auto-deletion.
+Failed backups will be checked and cleaned up during backupstore polling which is controlled by **Backupstore Poll Interval** setting.
+Hence this value determines the minimal wait interval of the cleanup. And the actual cleanup interval is multiple of **Backupstore Poll Interval**.
+			Disabling **Backupstore Poll Interval** also means to disable failed backup auto-deletion."
```
Comment (innobead) [inline chart/questions.yml]: Extra tab and space here

**Review by innobead — CHANGES_REQUESTED** [review]

File: chart/questions.yml (hunk 1 above)
Comment (innobead) [reply chart/questions.yml]: Did you use cherry-pick? It will help you just pick from the original commits.

Comment (mantissahz) [reply chart/questions.yml]: weird, I used cherry-pick.

Comment (mantissahz) [reply chart/questions.yml]: oh, I modified it at second commit.

File: enhancements/20220801-failed-backups-cleanup.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# Failed Backup Clean Up
```
Comment (innobead) [inline enhancements/20220801-failed-backups-cleanup.md:1]: Actually, enhance proposals no need to backport, but it's OK for sure.

**Review by innobead — APPROVED** [review]

File: chart/questions.yml (hunk 1 above)
Comment (innobead) [reply chart/questions.yml]: Yeah. The original fix was not in the right commit to cause this confusion.

**Review by innobead — APPROVED** [review]

**Pull request merged**